### ISSUE TITLE
Add ‘smart quotes’ as an option in preferences 

### ISSUE THREAD

**Issue by toddprouty** [scrod/nv#265]
Smart quotes or typographer’s quotes significantly improve the appearance of text, and it would be very useful to have this option. The only time smart quotes should be avoided is when writing code, but NV isn’t primarily a text editor for programmers. BBEdit and TextWrangler are good examples of how this can be done. One potential issue: this would likely need to be a global preference in NV, unlike the apps mentioned which can toggle the preference for individual documents.

**Comment by kimslawson** [comment]
Agreed on the need for this. In [another issue](https://github.com/scrod/nv/issues/120) closed mid-2011, the missing “Edit > Substitutions” menu was mentioned, with an assertion by @scrod that the toggle “Use Automatic Text Replacement” enables the default substitution behaviors. I am not sure this is true anymore, as I've tested it on my system running OS X 10.10.3 and Notational Velocity version 2.0 β5 (8) and it's not working. Can anyone confirm this?
